C#Backend/InpatientTherapySchedulingProgram/Services/RoomService.cs

```diff
@@ -0,0 +1,106 @@
+﻿using InpatientTherapySchedulingProgram.Models;
+using InpatientTherapySchedulingProgram.Services.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using InpatientTherapySchedulingProgram.Exceptions.RoomException;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace InpatientTherapySchedulingProgram.Services
+{
+    public class RoomService : IRoomService
+    {
+        private readonly CoreDbContext _context;
+
+        public RoomService(CoreDbContext context)
+        {
+            _context = context;
+        }
+
+        //Add Room
```

Comment:
Probably don't need floating comments here because the function name is descriptive enough.